flow/display_list.cc

```diff
@@ -1074,22 +1075,27 @@ void DisplayListBuilder::setBlendMode(SkBlendMode mode) {
 void DisplayListBuilder::setFilterQuality(SkFilterQuality quality) {
   Push<SetFilterQualityOp>(0, quality);
 }
-void DisplayListBuilder::setShader(sk_sp<SkShader> shader) {
+void DisplayListBuilder::setShader(const sk_sp<SkShader> shader) {
```

Comment:
I don't think this use of const does anything useful. If anything, it makes the `std::move(shader)` resort to a copy instead of a move. Did you perhaps mean `sk_sp<const SkShader> shader`?